frappe/website/report/website_analytics/website_analytics.py

```diff
@@ -56,33 +56,19 @@ def get_columns(self):
 		]
 
 	def get_data(self):
-		pg_query = """
-				SELECT
-					path,
-					COUNT(*) as count,
-					COUNT(CASE WHEN CAST(is_unique as Integer) = 1 THEN 1 END) as unique_count
-				FROM `tabWeb Page View`
-				WHERE  coalesce("tabWeb Page View".creation, '0001-01-01') BETWEEN %s AND %s
-				GROUP BY path
-				ORDER BY count desc
-			"""
-
-		mariadb_query = """
-				SELECT
-					path,
-					COUNT(*) as count,
-					COUNT(CASE WHEN is_unique = 1 THEN 1 END) as unique_count
-				FROM `tabWeb Page View`
-				WHERE creation BETWEEN %s AND %s
-				GROUP BY path
-				ORDER BY count desc
-			"""
-
-		data = frappe.db.multisql({
-				"mariadb": mariadb_query,
-				"postgres": pg_query
-			}, (self.filters.from_date, self.filters.to_date))
-		return data
+		Web_Page_View = frappe.qb.Table("Web Page View")
```

Comment:
```suggestion
		WebPageView = frappe.qb.Table("Web Page View")
```

Pythonic naming. Using Frappe's DocType controller names feels more natural to represent dt tables in these queries.